package.json

```diff
@@ -101,10 +101,10 @@
     "point-cluster": "^3.1.4",
     "polybooljs": "^1.2.0",
     "regl": "^1.3.11",
-    "regl-error2d": "^2.0.5",
-    "regl-line2d": "^3.0.12",
-    "regl-scatter2d": "^3.1.2",
-    "regl-splom": "^1.0.5",
+    "regl-error2d": "git://github.com/gl-vis/regl-error2d#0d27313d932fdb8ffaed17931b7258958bce0b88",
```

Comment:
@etpinard missing `npm` permission to publish this patch. Hopefully @dy would help?

Reply:
I don't have npm publish rights for `regl-error2d` either :cry:

Reply:
-> https://github.com/gl-vis/regl-error2d/issues/3 :crossed_fingers:

Reply:
@archmoj you should now have `npm publish` rights.

Reply:
I do have. Let me publish the new package. Thanks @etpinard.